WordPress/Classes/Models/WPAccount+Lookup.swift

```diff
@@ -15,6 +15,13 @@ public extension WPAccount {
         return self.uuid == uuid
     }
 
+    // This is here in an extension that belongs to the apps target so we can decouple WPAccount from AppConfiguration.
+    // Decoupling allows moving the type to WordPressData, see https://github.com/wordpress-mobile/WordPress-iOS/issues/24165.
+    @objc
+    static func tokenForUsername(_ username: String) -> String {
+        token(forUsername: username, isJetpack: AppConfiguration.isJetpack)
+    }
+
```

Comment:
Not sure whether the `+Lookup` extension is the best where to host this implementation, but it will do for now. "token for username" kind feels like looking up information, no?